main.go

```diff
@@ -164,7 +164,7 @@ func main() {
 		Clock:          clock,
 		DryRun:         *fDryRun,
 		KubeClient:     kubeClient,
-		KubectlClient:  kubectlClient,
+		KubeCtlClient:  kubeCtlClient,
```

Comment:
Is `KubeCtl`/`kubeCtl` the right casing? In my mind, `kubectl` is one word, even if it's a portmanteau of 'kubernetes' and 'control'.

Reply:
Agree - but most of "kubebuilder" is using this casing, so its easier to work being consistent with the lib.

Reply:
Okay, makes sense :+1: